app/command.go

```diff
@@ -646,15 +666,38 @@ func (a *App) UpdateCommand(oldCmd, updatedCmd *model.Command) (*model.Command,
 	updatedCmd.CreatorId = oldCmd.CreatorId
 	updatedCmd.TeamId = oldCmd.TeamId
 
-	return a.Srv().Store.Command().Update(updatedCmd)
+	command, err := a.Srv().Store.Command().Update(updatedCmd)
+	if err != nil {
+		var nfErr *store.ErrNotFound
+		var appErr *model.AppError
+		switch {
+		case errors.As(err, &nfErr):
```

Comment:
I don't see the `Update` method returning a NotFound error. But perhaps I have missed something ?